roles/convert2rhel/tasks/activation_keys_sca_enabled.yml

```diff
@@ -0,0 +1,48 @@
+---
```

Comment:
the only difference between this file, and the `_sca_disabled` one is, that this is not adding any subs to the keys, and the other one is, right?

Reply:
Exactly